kover-gradle-plugin/docs/configuring.md

```diff
@@ -29,11 +29,14 @@ koverReport {
   
     // verification rules for verification task
     verify {
+        // fail on verification error
+        warnOnFailure = false
```

Comment:
IMO, 'warnOnFailure' doesn't indicate that failure won't happen. I think something like `warningInsteadOfFailure` or conveys meaning better.

Reply:
Yeah, agreed